lib/client/rpc/balances.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.FutarchyRPCBalancesClient = void 0;
+const spl_token_1 = require("@solana/spl-token");
+const web3_js_1 = require("@solana/web3.js");
+class FutarchyRPCBalancesClient {
+    constructor(rpcProvider) {
+        this.rpcProvider = rpcProvider;
+    }
+    async fetchMainTokenWalletBalances(dao, ownerWallet) {
```

Comment:
Could we generalize this a bit more to accept single pub key or list of pub keys (or just create a new function).

Reply:
Added a TODO in the comments about this per our discussion.